testing/trino-benchto-benchmarks/README.md

```diff
@@ -84,10 +90,20 @@ tpch_medium: tpcds_10gb_txt
 
 With the scene set up as in the previous section, the benchmark can be run with:
 ```bash
-./mvnw clean package -pl :trino-benchto-benchmarks
-java -Xmx1g -jar trino-benchto-benchmarks/target/trino-benchto-benchmarks-*-executable.jar \
-    --sql trino-benchto-benchmarks/src/main/resources/sql \
-    --benchmarks trino-benchto-benchmarks/src/main/resources/benchmarks \
-    --activeBenchmarks=presto/tpch --profile=presto-devenv \
-    --overrides overrides.yaml
+# set JAVA_HOME to the JDK 8 home dir
+local_repo=$(./mvnw -B help:evaluate -Dexpression=settings.localRepository -q -DforceStdout)
+java -Xmx1g \
+            -jar "$local_repo/io/trino/benchto/benchto-driver/0.18/benchto-driver-0.18.jar" \
+            --profiles.directory "." \
+            --sql src/main/resources/sql \
+            --benchmarks src/main/resources/benchmarks \
+            --activeBenchmarks=presto/tpch \
+            --overrides "overrides.yaml" \
+            --timeLimit PT20M
 ```
+
+
+### Automated benchmarks
+
+A complete script which runs a small suite of benchmarks can be found in `perftest/benchto.sh`.
```

Comment:
This doesn't exist (yet).

Reply:
Isn't the script developed on the `benchmarks` branch where this PR is aimed at?

Reply:
Yes, but it should be separated from that effort so we can get this in faster.